smac/facade/smac_ac_facade.py

```diff
@@ -436,11 +439,8 @@ def __init__(self,
         if intensifier_kwargs is not None:
             intensifier_def_kwargs.update(intensifier_kwargs)
 
-        if intensifier is None:
-            intensifier_instance = (
-                Intensifier(**intensifier_def_kwargs)  # type: ignore[arg-type] # noqa F821
-            )  # type: AbstractRacer
-        elif inspect.isclass(intensifier):
+        assert intensifier is not None
```

Comment:
we do not need an assert statement here as it is made sure that the intensifier cannot be None at this point (and I would rather use a ValueError or sth), could you remove it, @filipbartek?

Reply:
I have removed the assertion.